slemicro/combustion/script

```diff
@@ -183,17 +228,23 @@ if [ "${KUBEVIP}" = true ]; then
 	        - name: vip_leaderelection
 	          value: "true"
 	        - name: vip_leaseduration
-	          value: "5"
+	          value: "30"
 	        - name: vip_renewdeadline
-	          value: "3"
+	          value: "20"
 	        - name: vip_retryperiod
-	          value: "1"
+	          value: "4"
 	        - name: address
 	          value: ${VIP}
-	        - name: prometheus_server
-	          value: :2112
+	EOF
+if [ ${CLUSTER} == "k3s" ]; then
```

Comment:
This needs to be tabbed I guess?
IDK if there is another way to do this, but it feels a little bit _dirty_ 😓 let me think about this.